test/utils/wasm_engine.hpp

```diff
@@ -10,6 +10,8 @@ namespace fizzy::test
 class WasmEngine
 {
 public:
+    using FuncRef = uintptr_t;
```

Comment:
Why `uintptr_t` and not `intptr_t`? I found no definitive answer.

Reply:
I wasn't thinking a lot about this. In the end `uintptr_t` helps in fizzy a bit because `uint32_t` -> `uintptr_t` conversion is lossless.

Reply:
Will be enough for wasm3?

Reply:
It is a pointer in wasm3 so I guess it doesn't matter too much.